specs/TrackingPrevention.md

```diff
@@ -0,0 +1,205 @@
+  /// against malicious trackers and most trackers across sites. Content and ads will likely have minimal personalization.
+  ///
+  /// This level blocks the most trackers but could cause some websites to not behave as expected.
+  ///
+  /// See [Current tracking prevention behavior](/microsoft-edge/web-platform/tracking-prevention#current-tracking-prevention-behavior)
+  /// for fine-grained information on what is being blocked with this level. This can change with different Edge versions.
+  COREWEBVIEW2_TRACKING_PREVENTION_LEVEL_STRICT,
+} COREWEBVIEW2_TRACKING_PREVENTION_LEVEL;
+
+/// Additional options used to create WebView2 Environment.
+[uuid(12e494a2-c876-11eb-b8bc-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2EnvironmentOptions3 : IUnknown {
+  /// The `IsTrackingPreventionEnabled` property is used to toggle tracking prevention feature in WebView2.
+  /// This property enables or disables tracking prevention for all the WebView2's created in the same environment.
+  /// By default this feature is enabled to block potentially harmful trackers and trackers from sites that
+  /// aren't visited before and set to `COREWEBVIEW2_TRACKING_PREVENTION_LEVEL_BALANCED`.
+  ///
+  /// You can set this property to false to disable the tracking prevention feature if the app only renders
+  /// content in the WebView2 that is known to be safe. Disabling this feature when creating environment also
+  /// improves runtime performance by skipping related code.
+  ///
+  /// You shouldn't disable this property if WebView2 is being used as a "full browser" with arbitrary navigation
+  /// and should protect end user privacy.
+  ///
+  ///
+  /// There is `TrackingPreventionLevel` property to control levels of tracking prevention of the WebView2's
+  /// associated with a same user profile. However, you can also turn off tracking prevention later using
+  /// `TrackingPreventionLevel` property and `COREWEBVIEW2_TRACKING_PREVENTION_LEVEL_OFF` value
+  /// but that doesn't improves runtime performance.
+  /// See `TrackingPreventionLevel` for more details.
+  ///
+  /// Tracking prevention protects users from online tracking by restricting
+  /// the ability of trackers to access browser-based storage as well as the network.
+  /// See [Tracking prevention](microsoft-edge/web-platform/tracking-prevention).
+  [propget] HRESULT IsTrackingPreventionEnabled([out, retval]  BOOL* value);
+  /// Sets the `IsTrackingPreventionEnabled` property.
+  [propput] HRESULT IsTrackingPreventionEnabled([in]  BOOL value);
+}
+
+/// This is the ICoreWebView2 profile.
+[uuid(ddc4070a-c873-11eb-b8bc-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2Profile5: IUnknown {
+  /// The `TrackingPreventionLevel` property sets the level of tracking prevention when `IsTrackingPreventionEnabled`
+  /// is enabled and otherwise level will not be updated and remains as `COREWEBVIEW2_TRACKING_PREVENTION_LEVEL_OFF`

```

Comment:
The property is actually modified (returns 'off') if the other property is false?

Reply:
See above